packages/xo-web/src/xo-app/xoa/support/index.js

```diff
@@ -29,15 +29,15 @@ const Support = decorate([
       checkXoa: async () => ({ stdoutCheckXoa: await checkXoa() }),
     },
     computed: {
-      stdoutSuppotTunnel: (_, { tunnelState }) =>
-        tunnelState !== undefined ? marked(tunnelState.stdout) : undefined,
+      stdoutSupportTunnel: (_, { tunnelState }) =>
+        tunnelState === undefined ? {} : { __html: marked(tunnelState.sdtout) },
```

Comment:
Is an empty object support?

Is it documented somewhere?

Is there a warning?

Reply:
It's not supported.
The `__html` key must be in the object.